### PR TITLE
1050 upstream aggregation patches

### PR DESCRIPTION
**Pull request by geissonator**
Confirmed that with these patches we can get /redfish/v1/AggregationService function working:

```
curl -k -H "X-Auth-Token: $token" -X GET https://${bmc}/redfish/v1/AggregationService
{
  "@odata.id": "/redfish/v1/AggregationService",
  "@odata.type": "#AggregationService.v1_0_1.AggregationService",
  "AggregationSources": {
    "@odata.id": "/redfish/v1/AggregationService/AggregationSources"
  },
  "Description": "Aggregation Service",
  "Id": "AggregationService",
  "Name": "Aggregation Service",
  "ServiceEnabled": true
}

curl -k -H "X-Auth-Token: $token" -X GET https://${bmc}/redfish/v1/AggregationService/AggregationSources
{
  "@odata.id": "/redfish/v1/AggregationService/AggregationSources",
  "@odata.type": "#AggregationSourceCollection.AggregationSourceCollection",
  "Members": [
    {
      "@odata.id": "/redfish/v1/AggregationService/AggregationSources/5B247A"
    }
  ],
  "Members@odata.count": 1,
  "Name": "Aggregation Source Collection"
}

curl -k -H "X-Auth-Token: $token" -X GET https://${bmc}/redfish/v1/AggregationService/AggregationSources/5B247A
{
  "@odata.id": "/redfish/v1/AggregationService/AggregationSources/5B247A",
  "@odata.type": "#AggregationSource.v1_3_1.AggregationSource",
  "HostName": "http://X.X.XX.XX:2280",
  "Id": "5B247A",
  "Name": "Aggregation source",
  "Password": null
}

```

These are all pulled directly from merged upstream patches.

### CODE REVIEW

File: redfish-core/lib/log_services.hpp [hunk 1]
```diff
@@ -1184,12 +1184,15 @@ inline void createDump(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
     std::vector<std::pair<std::string, std::variant<std::string, uint64_t>>>
         createDumpParamVec(createDumpParams);
 
-    createDumpParamVec.emplace_back(
-        "xyz.openbmc_project.Dump.Create.CreateParameters.OriginatorId",
-        req.session->clientIp);
-    createDumpParamVec.emplace_back(
-        "xyz.openbmc_project.Dump.Create.CreateParameters.OriginatorType",
-        "xyz.openbmc_project.Common.OriginatedBy.OriginatorTypes.Client");
+    if (req.session != nullptr)
```
Comment (gtmills) [inline redfish-core/lib/log_services.hpp:1187]: We want this change. Thanks.

File: subprojects/boost.wrap [hunk 2]
```diff
@@ -2,7 +2,7 @@
 directory = boost_1_81_0
 
 source_url = https://boostorg.jfrog.io/artifactory/main/release/1.81.0/source/boost_1_81_0.tar.bz2
-source_hash = 941c568e7ac79aa448ac28c98a5ec391fd1317170953c487bcf977c6ee6061ce
+source_hash = 71feeed900fbccca04a3b4f2f84a7c217186f28a940ed8b7ed4725986baf99fa
```
Comment (gtmills) [inline subprojects/boost.wrap:5]: Why change this ?

Comment (geissonator) [reply subprojects/boost.wrap:5]: I grabbed everything from Carson, which included https://gerrit.openbmc.org/c/openbmc/bmcweb/+/59799. Not sure why it started failing all of a sudden but we may just be getting lucky without this, using docker caches.

**Review by gtmills — APPROVED** [review]
These are all merged upstream.

**Pull request merged**